api_app/serializers.py

```diff
@@ -135,3 +181,81 @@ def create(self, validated_data):
             job.tags.set(tags)
 
         return job
+
+
+class FileAnalysisSerializer(_JobCreateSerializerMixin):
+    """
+    Job model's serializer for File Analysis.
+    Used for create()
+    """
+
+    tags_id = serializers.PrimaryKeyRelatedField(
+        many=True, write_only=True, queryset=Tag.objects.all()
+    )
+    runtime_configuration = serializers.JSONField(
+        required=False, default={}, write_only=True
+    )
+    file = serializers.FileField(required=True)
+    file_name = serializers.CharField(required=True)
+    is_sample = serializers.HiddenField(default=True)
+    file_mimetype = serializers.HiddenField(default=None)
+    analyzers_requested = serializers.ListField(default=list)
+
+    class Meta:
+        model = Job
+        fields = (
+            "id",
+            "source",
+            "is_sample",
+            "md5",
+            "force_privacy",
+            "disable_external_analyzers",
+            "file",
+            "file_name",
+            "file_mimetype",
+            "run_all_available_analyzers",
+            "runtime_configuration",
+            "analyzers_requested",
+            "tags_id",
+        )
+
+    def validate(self, attrs):
+        super().validate(attrs)
+
+        attrs["file_mimetype"] = calculate_mimetype(attrs["file"], attrs["file_name"])
+        return attrs
+
+
+class ObservableAnalysisSerializer(_JobCreateSerializerMixin):
+    """
+    Job model's serializer for Observable Analysis.
+    Used for create()
+    """
+
+    tags_id = serializers.PrimaryKeyRelatedField(
+        many=True, write_only=True, queryset=Tag.objects.all()
+    )
+    runtime_configuration = serializers.JSONField(
+        required=False, default={}, write_only=True
+    )
```

Comment:
Is it not possible to move these common fields to the `_JobCreateSerializerMixin` ?